src/components/Sidebar.tsx

```diff
@@ -101,12 +101,12 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = () => {
         {navigation.map(item => (
           <button
             key={item.name}
+            title={(item.name.charAt(0).toUpperCase() + item.name.slice(1))}
             onClick={() => setActiveNav(item.name as NavItemType)}
-            className={`flex text-sm border-l-2  ${
-              item.state()
-                ? 'text-white hover:text-gray-500 border-white'
-                : 'text-gray-500 hover:text-white border-gray-800'
-            } focus:outline-none border-box p-4`}
+            className={`flex text-sm border-l-2  ${item.state()
+              ? 'text-white hover:text-gray-500 border-white'
+              : 'text-gray-500 hover:text-white border-gray-800'
+              } focus:outline-none border-box p-4`}
```

Comment:
That change will (probably) cause problems on the eslint side, so please run `npm run lint:fix` script.

Reply:
Yep, it causes errors -> https://github.com/asyncapi/studio/runs/4919831435?check_suite_focus=true Please run mentioned script and commit changes and I will merge it :)

Reply:
Yeah, I have fixed it please review :)